src/concrete/ml/sklearn/base.py

```diff
@@ -65,6 +64,8 @@
 from hummingbird.ml import convert as hb_convert  # noqa: E402
 from hummingbird.ml.operator_converters import constants  # noqa: E402
 
+# Default number of bits to keep
+DEFAULT_ROUNDING_THRESHOLD_BITS = 6
```

Comment:
do we only use this here ? either way I would still put it in common/utils.py I think ! Also, how did we choose this value ? maybe comment a bit on why we chose 6 !

Reply:
I added a branch with some experiments (testing/impact_of_rouning_on_knn) in the PR's description.

Reply:
yes thanks for that, I'm just saying that if we add something like this in the code, it should probably be better to put in utils  instead + add some comments around it to explain why it was chosen like this !